### PR TITLE
[FIX] web_editor: open the powerbox in mobile

### PR DESCRIPTION
**Pull request by Goaman**
task-2751059

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by simongoffin** [comment]
Hello @dmo-odoo Are you ok with this?

**Comment by dmo-odoo** [comment]
@simongoffin The task has been tested in india and got a red flag from the tester as the powerbox. @Goaman is currently working on a more important bug but will have a look afterwards.

**Comment by Goaman** [comment]
@Zinston Could you have a look in order to r+ ?

File: addons/web_editor/static/lib/odoo-editor/src/powerbox/Powerbox.js [hunk 1]
```diff
@@ -142,12 +142,12 @@ export class Powerbox {
 
     addKeydownTrigger(triggerKey, options) {
         this.options.editable.addEventListener(
-            'keydown',
+            'input',
```
Comment (Zinston) [inline addons/web_editor/static/lib/odoo-editor/src/powerbox/Powerbox.js:145]: You should rename the function to match the change of event. I'm just wondering though if we do that if we can do it in stable since it might qualify as a change of API...
If not then don't bother doing it here but it should be done in master.

Did you test that it works properly with all keyboards (swiftkey?) and on all major browsers?

File: addons/web_editor/static/lib/odoo-editor/src/powerbox/Powerbox.js [hunk 2]
```diff
@@ -254,10 +260,12 @@ export class Powerbox {
...
-        this._currentValidate = () => {
-            const command = this._currentFilteredCommands.find(
-                c => c === this._currentSelectedCommand,
-            );
+        this._currentValidate = (command) => {
+            if (!command) {
+                command = this._currentFilteredCommands.find(
+                    c => c === this._currentSelectedCommand,
+                );
+            }
```
Comment (Zinston) [inline addons/web_editor/static/lib/odoo-editor/src/powerbox/Powerbox.js:268]: This part matches the commit message, the rest doesn't and should be with the previous commit.

Comment (Goaman) [reply addons/web_editor/static/lib/odoo-editor/src/powerbox/Powerbox.js:268]: true, solved

File: addons/web_editor/static/lib/odoo-editor/src/powerbox/Powerbox.js [hunk 3]
```diff
@@ -131,7 +131,7 @@ export class Powerbox {
                     ev => {
                         ev.preventDefault();
                         ev.stopImmediatePropagation();
-                        this._currentValidate();
+                        this._currentValidate(command);
```
Comment (Zinston) [inline addons/web_editor/static/lib/odoo-editor/src/powerbox/Powerbox.js:134]: This part belongs in the next commit. It was confusing for me to read the other commit, seeing you added an argument to the method but not where you use it.

File: addons/web_editor/static/lib/odoo-editor/src/powerbox/Powerbox.js [hunk 4]
```diff
@@ -158,6 +158,12 @@ export class Powerbox {
         );
     }
 
+    // todo: remove in master. It has been kept when changing addEventListener
+    // keydown to input in stable to avoid removing a public method.
```
Comment (Zinston) [inline addons/web_editor/static/lib/odoo-editor/src/powerbox/Powerbox.js:162]: While you're at it, I would rephrase to be more specific about the versions. I always find it strange to work in older versions and read things about "master". Which master is it?
So I'd say 15.0 instead of stable, and 15.3 instead of master (unless it's still time to merge in 15.2).

Comment (Goaman) [reply addons/web_editor/static/lib/odoo-editor/src/powerbox/Powerbox.js:162]: Master is the right name as whatever the version gets frozen, it will always be in master that we could remove the code.

File: addons/web_editor/static/lib/odoo-editor/src/powerbox/Powerbox.js (hunk 3 above)
Comment (Goaman) [reply addons/web_editor/static/lib/odoo-editor/src/powerbox/Powerbox.js:134]: done

File: addons/web_editor/static/lib/odoo-editor/src/powerbox/Powerbox.js (hunk 4 above)
Comment (Zinston) [reply addons/web_editor/static/lib/odoo-editor/src/powerbox/Powerbox.js:162]: I'm talking for posterity because this code will stay in 15.0 while master is not gonna remain the same. Anyway it doesn't matter enough to block this task, we can always eventually remove this comment.

**Comment by Zinston** [comment]
@robodoo r+

**Comment by Zinston** [comment]
@robodoo rebase-ff
